### PR TITLE
Added Options to Highlight, Fade and Delete the bars

### LINKED ISSUE #54: Delete a Bar on Long Click after showing an Alert Dialogue.
**Describe the solution you'd like**
A bar will get deleted by clicking on "yes" in an Alert dialogue Box which will appear after long Click on the selected Bar.
I want to work on this issue.

### LINKED ISSUE #74: Adding Highlight and Fade option in the Alert Dialogue to Highlight and Fade the bars.
Using radio buttons for adding options to highlight and fade the bars in the Alert Dialogue.
I want to work on this.

### PR DESCRIPTION
**Pull request by Alaska18**
This pull request addresses issues #74 and #54.

### CODE REVIEW

File: library/src/main/java/in/krharsh17/barview/BarGroup.java [hunk 1]
```diff
@@ -105,26 +111,26 @@
...
+            float progress,
+            int animationType,
+            int animationDuration,
+            int BAR_MARGIN,
+            int VERTICAL_SPACING,
+            int BAR_HEIGHT,
+            int LABEL_FONT_SIZE,
+            int VALUE_FONT_SIZE,
+            String labelTextColor,
+            String VALUE_TEXT_COLOR,
```
Comment (ShivangiSingh17) [inline library/src/main/java/in/krharsh17/barview/BarGroup.java:161]: what was the need to this? Earlier also there was a standard spacing.

**Comment by krharsh17** [comment]
@Alaska18 Could you please post a screenshot or a recording of the changes you've made?

**Comment by Alaska18** [comment]
These are the images.

**Comment by krharsh17** [comment]
Is there an option to choose the colour of highlight?

**Comment by Alaska18** [comment]
No. We have to code the color.

**Review by ShivangiSingh17 — DISMISSED** [review]

File: library/src/main/java/in/krharsh17/barview/BarGroup.java (hunk 1 above)
Comment (Alaska18) [reply library/src/main/java/in/krharsh17/barview/BarGroup.java:161]: That just happened while committing I think.

**Pull request closed without merging**